rero_ils/modules/patrons/api.py

```diff
@@ -775,6 +779,52 @@ def get_current_patron(record):
             if ptrn.organisation_pid == record.organisation_pid:
                 return ptrn
 
+    @classmethod
+    def set_communication_channel(
```

Comment:
Why not using this function in the alembic?

Reply:
This function does things both ways, for the migration we will not update the channel to 'EMAIL' if it has an email.